### PR TITLE
[fix-1921] use relative path when no fixture_path configured

### PR DESCRIPTION
**Pull request by 11factory**
related to issue https://github.com/rspec/rspec-rails/issues/1921

### CODE REVIEW

File: lib/rspec/rails/fixture_file_upload_support.rb [hunk 1]
```diff
@@ -27,9 +27,7 @@ def instance
           # to support Rails 3.0->3.1 using ActionController::TestCase class to resolve fixture_path
           # see https://apidock.com/rails/v3.0.0/ActionDispatch/TestProcess/fixture_file_upload
           def fixture_path=(value)
-            if ActionController::TestCase.respond_to?(:fixture_path)
-              ActionController::TestCase.fixture_path = value
-            end
+            ActionController::TestCase.fixture_path = value if ActionController::TestCase.respond_to?(:fixture_path)
```
Comment (JonRowe) [inline lib/rspec/rails/fixture_file_upload_support.rb]: Please undo this.

**Comment by 11factory** [comment]
@JonRowe changes have been done, CI seems already broken before my change

**Comment by JonRowe** [comment]
Sadly master is broken thats correct but your branch is more broken than master, so I'd really like to see a green build here! I'm trying to tackle master myself but it might take a while.

**Comment by JonRowe** [comment]
I have a green build over in #1950, you could consider rebasing your branch off that to see if your build goes green.

**Comment by JonRowe** [comment]
#1950 has now been merged so you should be able to get a green build if you rebase / merge master

**Comment by 11factory** [comment]
ping @JonRowe

**Pull request merged**